README.md

```diff
@@ -0,0 +1,117 @@
+# Web Test Transformer
+
+- [Web Test Transformer](#web-test-transformer)
+  - [Overview](#overview)
+  - [Setup](#setup)
+  - [Usage](#usage)
+    - [Dry Run](#dry-run)
+    - [CLI Flags](#cli-flags)
+      - [Required Flags](#required-flags)
+  - [Verification](#verification)
+    - [All PASS Requirement](#all-pass-requirement)
+  - [Issues](#issues)
+
+## Overview
+Web Test Transformer is designed to transform HTML tests in Chromium source from the legacy
+[js-test.js](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/web_tests/resources/js-test.js)
+framework to the current
+[testharness.js](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/web_tests/resources/testharness.js)
+framework.
+
+
+View the design document at [go/web-test-transformer](go/web-test-transformer) (internal Google access only).
+
+## Setup
+
+`$git clone git@github.com:Ecosystem-Infra/web-test-transformer.git`
+
+`$npm install` to install dependencies.
+
+## Usage
+
+Requirements:
+- You must be in `chromium/src/third_party/blink/web_tests` (for verification)
```

Comment:
nit: mention that this is to be run inside the chromium source tree for clarify -- even though the audience is blink eng, the doc is still in a public location.